data/language/en-GB.yml

```diff
@@ -2315,3 +2315,4 @@ strings:
   2260: "{COLOUR WINDOW_2} Length: {COLOUR BLACK}{INT32_1DP} tile(s)"
   2261: "{NEWLINE}{COLOUR WINDOW_2} Length: {COLOUR BLACK}{INT32_1DP} tile(s)"
   2262: "{NEWLINE}Length:  {INT32_1DP} tile(s)"
+  2263: "{COLOUR WINDOW_2}/199"
```

Comment:
You can get away with only updating en-GB all the other languages will fallback to using en-GB if there is no entry and as this is just displaying a number they will all be the same.